htdocs/fourn/class/fournisseur.product.class.php

```diff
@@ -954,7 +954,7 @@ public function display_price_product_fournisseur($showunitprice = 1, $showsupti
 			}
 			$out .= '</table>';
 		} else {
-			$out = ($showunitprice ? price($this->fourn_unitprice * (1 - $this->fourn_remise_percent / 100) + $this->fourn_remise).' '.$langs->trans("HT").' &nbsp; <span class="opacitymedium">(</span>' : '');
+			$out = ($showunitprice ? price($this->fourn_unitprice /* (1 - $this->fourn_remise_percent / 100) + $this->fourn_remise*/).' '.$langs->trans("HT").' &nbsp; <span class="opacitymedium">(</span>' : '');
```

Comment:
Why disabling the percent discount ?

Reply:
Because the discount is already into $this->fourn_unitprice and it was applied twice. The resuly was false

Reply:
So it is another bug fix. It seems this bug fix was already fixed by another PR so i close this PR.